enterprise/api_client/xpert_ai.py

```diff
@@ -0,0 +1,30 @@
+"""
+Xpert client
+"""
+
+import json
+
+import requests
+
+from django.conf import settings
+
+
+def chat_completion(prompt, role):
+    """
+    Generate response using xpert api.
```

Comment:
[Suggestion] You can describe arguments in docstring for better understanding and readability.
Like:
```
Arguments:
        prompt (str): {Description}
        role (str): {Description}
```